topics/metagenomics/tutorials/metagenomics-assembly/tutorial.md

```diff
@@ -0,0 +1,286 @@
+
+In order to construct a whole chromosome's sequence (ideally), reads are assembled into contigs and contigs into scaffolds. The main strategies used for genome and metagenome assembly are Greedy extension, Overlap Layout Consensus and De Bruijn charts. The following figure illustrates these strategies in brief. The nice paper {%cite miller2010%} on assemblers based on these algorithms will help you to better understand how they work.
+
+![Assembly algorithms](../../images/metagenomics-assembly/assembly-algorithms.png){:width="70%"}
+
+Image from {%cite carpentries%}
+
+Different assemblers have different computational characteristics. Their performance varies according to the microbiome. The preference of one assembler over another depends on the purpose at hand. It was proven by the first round of the Critical Assessment of Metagenome Interpretation initiative ({%cite sczyrba2017%}) and confirmed by the second round of the CAMI initiative ({%cite meyer2022%}, {%cite meyer2021%}). Among the most popular are MetaSPAdes ({%cite nurk2017%}) and Megahit ({%cite li2015%}). 
+
+### Data
+
+We will use the [Temporal shotgun metagenomic dissection of the coffee fermentation ecosystem](https://www.ebi.ac.uk/metagenomics/studies/MGYS00005630#overview).
+
+Based on the real dataset of the coffee fermentation system, we generated mock datasets for this tutorial.
+
+For an in-depth analysis of the structure and functions of the coffee microbiome, a temporal shotgun metagenomic study (six time points) was performed.
+
+Real datasets are available for download in [ENA](https://www.ebi.ac.uk/ena/browser/view/PRJEB24129?show=reads), as well as metadata about samples. There are six samples of a coffee fermentation system sequenced with Illumina MiSeq utilizing whole genome sequencing. This dataset contains paired-end reads.
+
+#### Upload dataset to Galaxy
+First we need to get some data into our history. There are different ways to upload data into the Galaxy. You can upload files from your computer, or Galaxy can also fetch data directly from external sources.
+
+In case of a not very large dataset it’s more convenient to upload data directly from your computer to Galaxy. In case of a large dataset, we can use FTP server or the [Galaxy Rule-based Uploader]({% link topics/galaxy-interface/tutorials/upload-rules/tutorial.md %}).
+
+> ### {% icon hands_on %} Hands-on: Upload data into Galaxy
+>
+> 1. Create new history
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. At the top of the Tools panel (on the left), click “Upload Data”
+> ![Upload data](../../images/metagenomics-assembly/upload-data.png){:width="40%"}
```

Comment:
To avoid duplication, please use the "import via url" snippet and remove the screenshots

`{% snippet  faqs/galaxy/datasets_import_via_link.md %}`